### DISCUSSION TITLE
Is there an internal function to get all theme options as an array?

### DISCUSSION THREAD

**Discussion by afagard** [htmlburger/carbon-fields#1033] [Q&A]
Is there an internal function to get all theme options as an array? I want to load it in to a Tightenco Collection.

**Comment by jorostoyanov** [comment]
Hi @afagard 

No, there isn't such helper function. You should manually create the array

**Reply by afagard** [reply]
is there a more efficient way then doing it like so: `$wpdb->get_results("SELECT * FROM 'wp_options' WHERE 'option_name' LIKE '%_crb%'")`?

**Reply by jorostoyanov** [reply]
I would only suggest to use WP Cache when loading all options
